app/javascript/mastodon/features/explore/links.jsx

```diff
@@ -35,7 +35,7 @@ class Links extends PureComponent {
 
     const banner = (
       <DismissableBanner id='explore/links'>
-        <FormattedMessage id='dismissable_banner.explore_links' defaultMessage='These news stories are being talked about by people on this and other servers of the decentralized network right now.' />
+        <FormattedMessage id='dismissable_banner.explore_links' defaultMessage='These are news stories being shared the most on the social web today. Newer news stories posted by more different people are ranked higher.' />
```

Comment:
I'm slightly afraid this may be misleading people into thinking this is what's trending on the *whole* fediverse when it's “only” the posts that are known to your own server (which is going to be a huge difference for small-ish servers), but this was already a concern with the previous wording, and it's difficult striking a good balance between concise and understandable and precise enough.